### PR TITLE
adding sass, gulp, browsersync

### PR DESCRIPTION
**Pull request by andrewmartin**
@thatmartinguy 

Thought I'd help you with some better tools.

After you merge this in, go here and install node:
https://nodejs.org/en/download

Install that, then in the project root in the terminal, type:

`npm run setup`

Then, wait.

After it does everything, just type `gulp` in the command line, a browser window will open with livereload.

From now on you'll need to edit the `sass/main.scss` file for your changes, as `gulp` will generate and build out the `css/main.css` file from now on... Keep in mind you can write pure CSS in here, SCSS/SASS are basically the same thing as CSS.

TL;DR –– install node, type `npm run setup` in the command line...wait...then, after that's done, from now on type `gulp` in the command line and make edits only to scss/main.scss file from now on to develop.

Hope this helps.

### CODE REVIEW

**Comment by andrewmartin** [comment]
Also, some changes i'd recommend:

1. you have a 14MB image file in there...you definitely don't want to host that...
2. Don't use `vh` as a font size, way too complicated for this project...just use `px` font sizes instead. I still use `px` on almost all projects with success
3. Keep an eye on your terminal as you are working with the SASS file, it'll throw errors if you type bad CSS/SASS
4. I loaded in the fontawesome fonts for you, always check the Chrome console for errors, you were also not loading in the `tether` script that's required by Bootstrap.

I don't want to hold your hand through this, but I think this will get you up and running with some speedier development, hoping you can figure it out. Good luck.
